src/main/java/com/ita/if103java/ims/dao/impl/UserDaoImpl.java

```diff
@@ -286,8 +293,9 @@ private PreparedStatement getPreparedStatement(User user, Connection connection)
         public static final String SQL_SELECT_ALL_USERS = """
                 SELECT *
                 FROM users
-                Limit ?
-                Offset ?
+                ORDER BY %s
+                Limit %s
+                Offset %S
```

Comment:
use ? instead of %s

Reply:
I use %s to make `string.format`. I will change it